faculty/clients/project.py

```diff
@@ -115,12 +134,32 @@ def list_accessible_by_user(self, user_id):
         endpoint = "/user/{}".format(user_id)
         return self._get(endpoint, _ProjectSchema(many=True))
 
+    def list_all(self, include_archived=False):
+        """List all projects on the Faculty deployment.
+
+        This method requires administrative privileges not available to most
+        users.
+
+        Parameters
+        ----------
+        include_archived : bool, optional
+            If True, return archived projects. Default: False.
+
+        Returns
+        -------
+        List[Project]
+            The projects in Faculty.
+        """
+        params = {"includeArchived": int(include_archived)}
+        return self._get("/project", _ProjectSchema(many=True), params=params)
```

Comment:
AFAIK, the backend accepts something like `GET /project?includeArchived=true`. Is this the resulting URL? Is there something about some bool/int conversion I'm missing?

Reply:
The backend accepts "true", "false", "1" or "0", so casting the bools to ints seemed most pragmatic.

Reply:
Oh, I didn't know Play worked that way. 👍